client/components/RoomListItem.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+const RoomListItem = React.createClass({
+  mixins: [ReactMeteorData],
+
+  getMeteorData () {
+    const messagesHandle = Meteor.subscribe("chatterMessages", {
+      roomId: this.props.room._id,
+      messageLimit: 30
+    });
+
+    const countHandle = Meteor.subscribe("chatterUserRooms", {
+      roomId: this.props.room._id
+    });
+
+    const subsReady = messagesHandle.ready() && countHandle.ready();
+
+    let message = "no messages yet",
```

Comment:
prefer multiple let statements instead of comma separated list to avoid accidentally global